Analysis/output/country_data_report.Rmd

```diff
@@ -18,14 +25,14 @@ library(taxdat)
 library(sf)
 ```
 
-# Mapping results summary for `r stringr::str_extract(params$config, "(?<=g_)[A-Z]{3}(?=_)")` and period `r stringr::str_extract(params$config, "(?<=_)[0-9]{4}_[0-9]{4}(?=\\.)")`
+# Mapping results summary for `r stringr::str_extract(params$config, "[A-Z]{3}")` and period `r stringr::str_extract(params$config, "[0-9]{4}_[0-9]{4}")`
 
 ## Cholera input data
 **Data input:**
-```{r cases, echo=FALSE, message=FALSE, warning=FALSE}
+```{r cases}
```

Comment:
Any reason this got removed?

Reply:
they are already in the knitr::opts_chunk$set